springql-core/src/expression/module.md

```diff
@@ -0,0 +1,9 @@
+# module expression
```

Comment:
[IMO] How about removing section titles?

In this PR, just copy-and-paste the original doc comments would be fine.